types/src/grants.d.ts

```diff
@@ -18,6 +19,25 @@ export type GrantMetadata = {
   name: string;
   description: string;
   logoURI?: string;
+  quote?: string;
+  lastUpdated?: string;
+  projectWebsite?: string;
+  projectGithub?: string;
+  twitterHandle?: string;
```

Comment:
Two comments:
1. I thought we decided on a Twitter URL for consistency instead of the handle, but maybe I'm misremembering?
2. Perhaps `websiteURI`, `githubURI` and `twitterURI` for consistency with `logoURI` in naming conventions?

Reply:
I took these names from the issue @apbendi created, do we want to change these?

Reply:
My understanding is that this is the type we want to use for this PR

```typescript
type GrantMetadata = {
  name: string;
  description: string;
  logoURI?: string;
  properties: {
    websiteURI?: string;
    githubURI?: string;
    twitterURI?: string;
    [key: string]?: string // for extensions/flexibility
  }
}
```

Reply:
Awesome, I've moved to this definition:

```ts
export type GrantMetadata = {
  name: string;
  description: string;
  logoURI?: string;
  properties: {
    websiteURI?: string;
    githubURI?: string;
    twitterURI?: string;
  } & {
    [key: string]: string; // for extensions/flexibility
  };
};
```